### ISSUE TITLE
Add possibility to copy/paste 'custom properties' of tiles

### ISSUE THREAD

**Issue by igroglaz** [bjorn/tiled#2171]
It would be great if there would be possibility to copy 'custom properties' of tiles, eg

1) user select row of tiles in Tiled @ .tsx file
2) RMB -> "Copy 'custom properties'"
3) user select a row where to paste 'custom properties'

I've got huge tileset (2000+) tiles and I map it properties directly in Tiled (to draw maps in .txt file afterward). Such feature would help a lot in operation big tilesets :+1:

**Comment by bjorn** [comment]
Actually this already works, but the Copy and Paste actions are not in the context menu of the tiles, but rather in the context menu of the Properties view. Before copying you can select multiple custom properties, or all of them if you want to copy them all.

**Comment by igroglaz** [comment]
Yep, thanks :) But there is a possibility to copy only one tile's properties to another tile/tiles. 

/rfe is, eg:

User selects 3 tiles in Tiled @ .tsx file: tile A, tile B, tile C.

Tile / Property
A / 0
B / 1
C / 2

We want to copy properties of tiles A, B, C to D, E, F; so they would have the same property values.

Right now we could only copy one-by-one - copy tile A to tile D, tile B to tile E... It would be great to be able to select all this three tile and copy their values to another three tile in one go :)

**Comment by bjorn** [comment]
Hmm, I'm not sure if this action would be generally useful or intuitive. What happens if the user selects a different number of tiles when copying than when pasting?

It sounds to be more like something that you could solve by writing a small script to copy those custom properties over. To try this, install the Tiled development snapshot and see the [scripting documentation](http://doc.mapeditor.org/en/latest/reference/scripting/) for how to read/write custom properties.

**Comment by igroglaz** [comment]
It is super useful :) Eg:

For mapping purposes I got tiles a-z for first row of tileset and A-Z for second row (it's 255 ANSI; so my tileset coords looks like 'zebra'). Without this feature I have to map every 2nd row manually (and I got 128 rows..) which is pretty dull.

> What happens if the user selects a different number of tiles when copying than when pasting?

Then it would work like in Excel - it would paste as much as it fits.

**Comment by bjorn** [comment]
I still think you're better off scripting this then waiting for me or somebody else to get around to it. If you need help with that just ask.
